docker/app/config/fpm-pool.conf

```diff
@@ -53,4 +53,7 @@ catch_workers_output = yes
 decorate_workers_output = no
 
 ; Enable ping page to use in healthcheck
-ping.path = /fpm-ping
\ No newline at end of file
+ping.path = /fpm-ping
+
+user = abc
```

Comment:
Is there a reason why these are called "abc'? :)

Reply:
`abc` is the non-root user LSIO assigns PUID/PGID too.